scripts/create-kubestellar-demo-env.sh

```diff
@@ -116,6 +116,7 @@ for cluster in "${clusters[@]}"; do
      echo -e "Creating cluster ${cluster}..."
      kind create cluster --name "${cluster}" >/dev/null 2>&1 &&
      kubectl config rename-context "kind-${cluster}" "${cluster}" >/dev/null 2>&1
+     kubectl config rename-context "kind-${cluster}" "${cluster}" >/dev/null 2>&1
```

Comment:
It is not safe to just repeat this command! For example:

```console
mspreitz@mjs13 kubestellar % kubectl config rename-context kind-cluster1 cluster1
error: cannot rename the context "kind-cluster1", it's not in /Users/mspreitz/.kube/config
```

FYI, I asked (in the `#sig-cli` channel of the Kubernetes Slack organization) about concurrent `kubectl config rename-context` commands, and was told:

> so it's safe in the sense that it won't create a broken state from reading at different versions before mutating, but one of the commands might fail

> so it's not safe to assume they succeeded. but it is safe to assume that your configs won't be mangled.

See https://kubernetes.slack.com/archives/C2GL57FJ4/p1729873069405909 , while you still can, for the whole conversation.

I tried testing, and there is something strange going on. I got no output from the `kind create` commands in this loop.

Given that concurrent context renaming is problematic, and there is not much time to be saved there, and it is being lost anyway by repeating commands, I think the better approach is to do the context renamings serially, after waiting for the cluster creation commands to complete.

Reply:
Here is a revised version of this section that I have successfully tested.

```shell
for cluster in "${clusters[@]}"; do
   (
     echo -e "Creating cluster ${cluster}..."
     kind create cluster --name "${cluster}" >/dev/null 2>&1
     echo -e "${cluster} creation and context setup complete"
   ) &
done
wait
kubectl config rename-context kind-cluster1 cluster1
kubectl config rename-context kind-cluster2 cluster2
```

Reply:
See #2546